dapr/clients/grpc/client.py

```diff
@@ -284,6 +284,56 @@ def publish_event(
             headers=call.initial_metadata(),
             trailers=call.trailing_metadata())
 
+    def get_state(
+        self,
+        store_name: str,
+        key: str
+    ):
+        try:
+            if len(store_name) == 0 or len(store_name.strip()) == 0:
+                raise ValueError("State store name cannot be empty")
+            req = api_v1.GetStateRequest(store_name=store_name, key=key)
+            resp = self._stub.GetState(req)
+            return resp
+        except Exception as e:
+            return e
+
+    def save_state(
+        self,
+        store_name: str,
+        key: str,
+        value: object,
+        etag: str = "",
+        state_options: common_v1.StateOptions = None
+    ):
+        """Saves key-value pairs to a statestore
+        The example saves states to a statestore:
+            from dapr import DaprClient
+            with DaprClient() as d:
+                resp = d.save_state(
+                    store_name='state_store'
+                    states=[{'key': 'key1', 'value': 'value1'}],
+                )
+        Args:
+            store_name (str): the state store name to save to
+            states (List[dict]): the key-value pairs to be saved
+        Returns:
+            None
+        """
+        try:
+            if len(store_name) == 0 or len(store_name.strip()) == 0:
+                raise ValueError("State store name cannot be empty")
+            state = [common_v1.StateItem(
+                key=state['key'], value=value.encode('utf-8'), etag=etag.encode('utf-8'),
+                options=state_options)]
+            if state_options is not None:
+                state.options = state_options
+            req = api_v1.SaveStateRequest(store_name=store_name, states=state)
+            response = self._stub.SaveState(req)
+            return response
```

Comment:
all api needs to return DaprResponse object which includes response headers from dapr runtime.

Reply:
I noticed for `publish_event`, it's
```
        response, call = self._stub.PublishEvent.with_call(req, metadata=metadata)

        return DaprResponse(
            headers=call.initial_metadata(),
            trailers=call.trailing_metadata())
```
Do I implement
```
            req = api_v1.SaveStateRequest(store_name=store_name, states=state)
            response, call = self._stub.SaveState.with_call(req, metadata=metadata)
        return DaprResponse(
            headers=call.initial_metadata(),
            trailers=call.trailing_metadata())

```
?

Reply:
yes you need to do. I merged my pr. please check the latest code. I removed trailer part from response.